app/models/draw_suites_update.rb

```diff
@@ -105,4 +104,27 @@ def error(error)
       msg: { error: "Suites update failed: #{error}" }
     }
   end
+
+  # create dynamic attr_readers based on available suite sizes
+  def method_missing(method_name, *args, &block)
+    super unless valid_method_names.include? method_name
+    instance_variable_get("@#{method_name}")
+  end
+
+  def valid_method_names
+    @valid_method_names ||= all_suite_sizes.map { |s| params_for(s) }.flatten
+  end
+
+  def all_suite_sizes
+    @all_suite_sizes ||= SuiteSizesQuery.new(Suite.available).call
+  end
+
+  def params_for(size)
+    param_headings = %w(suite_ids drawn_ids drawless_ids)
+    param_headings.map { |p| "#{p}_#{size}".to_sym }
+  end
+
+  def respond_to_missing?(method_name, include_all = false)
+    valid_method_names.include?(method_name) || super
```

Comment:
It hits the default behavior in `Object` I believe, which is `false` unless it shows up in the Ruby method inheritance chain.